spec/60-trace-id-format.md

```diff
@@ -73,3 +74,9 @@ and receive an additional flag. Compliant systems will propagate this flag along
 all other key/value pairs. Existing systems which are not capable of
 `tracestate` propagation will truncate all additional values from `tracestate`
 and only pass along that flag.
+
+* This section provides guidance for dealing with short `trace-id`s generated by
```

Comment:
I think this point is wrong. we are not providing guidance for accepting short trace ids from non-compliant systems, but rather for systems which want to be compliant but can only process an 8 byte id.

@SergeyKanzhelev